editor/scene_tree_dock.cpp

```diff
@@ -1625,8 +1625,21 @@ void SceneTreeDock::_update_script_button() {
 			button_clear_script->show();
 		}
 	} else {
-		button_create_script->show();
-		button_clear_script->hide();
+		if (EditorNode::get_singleton()->get_editor_selection()->get_selection().size() == 0) {
```

Comment:
`else if` would be better, and maybe put the `0` case first, then `1`, then `else` for readability.

Reply:
Done